src/main/java/life/qbic/domain/sample/Sample.java

```diff
@@ -186,6 +192,14 @@ public static class CurrentState {
     public Status status() {
       return status;
     }
+
+    /**
+     * The instant from which the current state is valid from.
```

Comment:
NIT: we should call it validSince to conform to the name used as a String in the other class(es)

Reply:
good catch